.github/workflows/ci-and-e2e-tests.yaml

```diff
@@ -0,0 +1,35 @@
+name: CI & E2E Tests
+
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
I got my answer from the previous PR :slightly_smiling_face: 
I think we should just exclude the protected branches. No need to re-run tests: it's already done by bert-e or in the PR before merging. Similar to other repos.

Reply:
No bert-e will be added in this repo, answered in the other PR :)